zipline/data/bundles/core.py

```diff
@@ -38,39 +37,59 @@
 
 def asset_db_path(bundle_name, timestr, environ=None):
     return pth.data_path(
-        [bundle_name, timestr, 'assets-%d.sqlite' % ASSET_DB_VERSION],
-        environ=environ,
+        list(asset_db_relative(bundle_name, timestr, environ)),
```

Comment:
Not necessarily, I made these a list so the underlying logic of paths.py wouldn't be changed